common/engine/keyboardprocessor/debian/watch

```diff
@@ -1,2 +1,2 @@
 version=4
-https://downloads.keyman.com/linux/(?:beta|stable)/(\d\S+)/keyman-keyboardprocessor-(.+)\.tar\.gz debian uupdate
+https://downloads.keyman.com/linux/(?:$tier)/(\d\S+)/keyman-keyboardprocessor-(.+)\.tar\.gz debian uupdate
```

Comment:
> .../(?:$tier)/...

Does this work? I didn't know you can use variables here. Oh, I see, it gets replaced by `launchpad.sh` (I wonder if a comment would be helpful?)

There's probably no need to use a regex now. Just `.../$tier/...` should work as well I'd think.

Reply:
k, I've added a comment to the watch files and removed the regex